lib/hal/video.c

```diff
@@ -227,6 +233,22 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 		if(pVidMode->refresh != refresh)
 			continue;
 
+		if(dwAdapter == AV_PACK_HDTV)
+		{
+			is_480p = pVidMode->dwMode & 0x80000000;
```

Comment:
That is not what that bit means.
It is also set for 720p, 1080i and the VGA modes.

I'm not sure what exactly it stands for, but it's definitely not just `is_480p`.

My best naming was `is_hd`, but it's still bad.

Reply:
Name suggestion: `is_progressiveScan`?

Reply:
I went with `is_hd` for now.